source/acquaman/BioXASMain/BioXASMainXASScanConfiguration.cpp

```diff
@@ -0,0 +1,73 @@
+#include "BioXASMainXASScanConfiguration.h"
+
+#include "dataman/AMScanAxisEXAFSRegion.h"
+#include "acquaman/BioXASMain/BioXASMainXASScanActionController.h"
+#include "ui/BioXAS/BioXASMainXASScanConfigurationView.h"
+
+BioXASMainXASScanConfiguration::BioXASMainXASScanConfiguration(QObject *parent) : AMStepScanConfiguration(parent)
+{
+    setName("Unnamed Scan");
+    setUserScanName("Unnamed Scan");
+
+    edgeEnergy_ = -1;
+
+    AMScanAxisEXAFSRegion *region = new AMScanAxisEXAFSRegion;
+    AMScanAxis *axis = new AMScanAxis(AMScanAxis::StepAxis, region);
+    appendScanAxis(axis);
+}
+
+BioXASMainXASScanConfiguration::BioXASMainXASScanConfiguration(const BioXASMainXASScanConfiguration &original)
+    : AMStepScanConfiguration(original)
+{
+    setName(original.name());
+    setUserScanName(original.userScanName());
```

Comment:
You'll want to set the userScanName to name (```setUserScanName(original.name());```) because the user scan name is actually not used consistently.  Behaviour will be more what you'd expect with the above suggestion.